srcpkgs/python3-libclang-py3/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-libclang-py3'
+pkgname=python3-libclang-py3
```

Comment:
Check Manual.md for naming conventions for language binding packages. I think it should be `libclang-python3`, but @asergi may know better.

Reply:
Changed it and make a new PR. Thanks